var/spack/repos/builtin/packages/sst-elements/package.py

```diff
@@ -159,6 +163,9 @@ def configure_args(self):
         if "+otf" in spec:
             args.append("--with-otf=%s" % spec["otf"].prefix)
 
+        if "+ariel_mpi" in spec:
```

Comment:
(Optional follow-up) Recommended syntax for this is now
```suggestion
        if spec.satisfies("+ariel_mpi"):
```
The behavior is largely the same but allows better detection of those corner cases where the `in` strategy fails.